src/Driver/AbstractDB2Driver.php

```diff
@@ -64,4 +66,38 @@ public function getSchemaManager(Connection $conn)
     {
         return new DB2IBMiSchemaManager($conn);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function createDatabasePlatformForVersion($version)
+    {
+        if (version_compare($this->getVersionNumber($version), '7.3', '>=')) {
```

Comment:
Replace by the version exposed for [7.2 TR3](https://www.ibm.com/support/pages/ibm-i-72-tr3-enhancements).